controller/src/test/java/io/pravega/controller/server/eventProcessor/ControllerEventProcessorPravegaTablesStreamTest.java

```diff
@@ -15,18 +15,116 @@
  */
 package io.pravega.controller.server.eventProcessor;
 
+import io.pravega.client.stream.ScalingPolicy;
+import io.pravega.client.stream.StreamConfiguration;
+import io.pravega.common.Exceptions;
+import io.pravega.common.concurrent.Futures;
+import io.pravega.controller.mocks.EventHelperMock;
+import io.pravega.controller.mocks.EventStreamWriterMock;
 import io.pravega.controller.mocks.SegmentHelperMock;
+import io.pravega.controller.server.SegmentHelper;
+import io.pravega.controller.server.eventProcessor.requesthandlers.CommitRequestHandler;
 import io.pravega.controller.server.security.auth.GrpcAuthHelper;
+import io.pravega.controller.store.VersionedMetadata;
 import io.pravega.controller.store.stream.StreamMetadataStore;
+import io.pravega.controller.store.stream.AbstractStreamMetadataStore;
+import io.pravega.controller.store.stream.PravegaTablesStreamMetadataStore;
+import io.pravega.controller.store.stream.VersionedTransactionData;
+import io.pravega.controller.store.stream.State;
+import io.pravega.controller.store.stream.OperationContext;
+import io.pravega.controller.store.stream.TxnStatus;
 import io.pravega.controller.store.stream.StreamStoreFactory;
+import io.pravega.controller.store.PravegaTablesStoreHelper;
+import io.pravega.controller.store.stream.records.CommittingTransactionsRecord;
+import io.pravega.controller.store.task.TaskStoreFactory;
+import io.pravega.controller.task.EventHelper;
+import io.pravega.controller.task.Stream.StreamMetadataTasks;
+import io.pravega.controller.task.Stream.StreamTransactionMetadataTasks;
+import io.pravega.controller.util.Config;
+import io.pravega.shared.controller.event.CommitEvent;
+import io.pravega.test.common.AssertExtensions;
+import org.junit.Test;
+import  static org.junit.Assert.assertFalse;
```

Comment:
These 2 static imports have an extra initial space and an unrecognized character by Git (perhaps due to the editor used?)

Reply:
fixed

Reply:
> System test run on PR branch completed successfully without any errors. Checked Controller logs for NPE exceptions and could not find any. Will do multiple system test runs to confirm this is fixed since the issue is a super corner case and may not be hit on most occasions.

So far completed 5 system test runs on the PR branch and none had any NullPointerExceptions in Controller logs.